product_security/red_hat_responsibilities/dedicated_network_security.adoc

```diff
@@ -0,0 +1,105 @@
+[[dedicated-network-security]]
+= Network Security
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+[[network-segregation]]
+== Network Segregation
+
+{product-title} uses a software-defined networking (SDN) approach to provide a
+unified cluster network that enables communication between pods across the
+cluster. This pod network is established and maintained by the OpenShift SDN,
+which configures an overlay network using Open vSwitch (OVS).
+
+By default, {product-title} uses the *ovs-networkpolicy* plug-in, which allows
+project administrators to configure their own isolation policies using
+NetworkPolicy objects. These policies can be fully managed by project owners and
+`dedicated-admin` users to add network-level isolation for the projects.
+
+Additionally, some {product-title} clusters also support the *ovs-multitenant*
+plug-in, which provides project-level isolation for pods and services. Each
+project receives a unique Virtual Network ID (VNID) that identifies traffic from
+pods assigned to the project. By default, pods from different projects cannot
+send packets to or receive packets from pods and services of a different
+project. However, the cluster administrator can join projects and later isolate
+them using admin commands. For more information on pod network management, see
+xref:../../admin_guide/managing_networking.adoc#admin-guide-manage-networking[Managing
+Networking].
+
+[[network-segregation-security-groups]]
+=== Security Groups
+
+Security groups are provided by the cloud provider. They provide security at the
+protocol and port access level. Each security group works similar to a firewall
+and has a set of rules that filter traffic coming into and out of a node.
+
+{product-title} has separate security groups configured for the following
+components:
+
+* master nodes
+* infrastructure nodes
+* compute Nodes
+* etcd
+* router ELB
+* External master ELB
+* Internal master ELB
+
+[[encryption-for-data-at-rest]]
+== Encryption for Data at Rest
+
+All new persistent volumes (PVs) are encrypted at rest. PVs are backed by block
+storage, which is Read-Write-Once and will be dynamically provisioned and
+recycled based on application requests.
+
+[[encryption-for-data-in-transit]]
+== Encryption for Data in Transit
+
+All communication channels with the REST API, as well as between master
+components such as etcd and the API server, are secured with TLS. TLS provides
+strong encryption, data integrity, and authentication of servers with X.509
+server certificates and public key infrastructure. By default, a new internal
+PKI is created for each deployment of {product-title}. The internal PKI uses
+2048 bit RSA keys and SHA-256 signatures.
+
+The insecure versions SSL 2.0 and SSL 3.0 are unsupported and not available. The
```

Comment:
Replace lines 73 to remove SSL reference.
Replace Line 74 with The
{product-title} server and `oc` client provides TLS 1.2 or newer by default.